packages/meditrak-server/src/tests/apiV2/surveyResponse.test.js

```diff
@@ -441,6 +429,57 @@ describe('surveyResponse endpoint', () => {
     expect(moment(dbResponse.end_time).isSame('2021-01-01T23:59:59.000Z')).to.be.true;
   });
 
+  describe('non-periodic', () => {
+    it('Should throw if a survey has no answers', async () => {
+      const response = await app.post('surveyResponse', {
+        body: {
+          survey_id: surveyId,
+          entity_id: ENTITY_ID,
+          timestamp: 123,
+          answers: {},
+        },
+      });
+
+      expectError(response, /must contain at least one answer/);
```

Comment:
Looks like this test is failing for some reason?

Reply:
It was some test cross-contamination due to reusing the same test survey code, fixed